streaming-binlog/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,26 @@
+#// Copyright 2020 Google LLC
+#//
+#// Licensed under the Apache License, Version 2.0 (the "License");
+#// you may not use this file except in compliance with the License.
+#// You may obtain a copy of the License at
+#//
+#//      http://www.apache.org/licenses/LICENSE-2.0
+#//
+#// Unless required by applicable law or agreed to in writing, software
+#// distributed under the License is distributed on an "AS IS" BASIS,
+#// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#// See the License for the specific language governing permissions and
+#// limitations under the License.
+
+# Direct log messages to stdout
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
```

Comment:
Is there a benefit for preferring `log4j.properties` over `log4j2.xml` which is used in the other modules/sub-projects?

Reply:
Nice catch, I'll change to version2

Reply:
I have changed this!